nailgun/entities.py

```diff
@@ -3811,6 +3811,32 @@ def list_smart_variables(self, synchronous=True, **kwargs):
         return _handle_response(response, self._server_config, synchronous)
 
 
+class Package(Entity, EntityReadMixin, EntitySearchMixin):
+    """A representation of a Package entity."""
+
+    def __init__(self, server_config=None, **kwargs):
+        self._fields = {
+            'arch': entity_fields.StringField(),
+            'checksum': entity_fields.StringField(),
+            'description': entity_fields.StringField(),
+            'epoch': entity_fields.StringField(),
+            'filename': entity_fields.StringField(),
+            'name': entity_fields.StringField(
+                str_type='alpha',
+                length=(6, 12)
```

Comment:
I think `lenth` is used when creating entity, for read-only entities there is no use in this.